nuxeo-coldstorage/src/main/resources/schemas/coldstorage.xsd

```diff
@@ -8,5 +8,6 @@
   <xs:element name="coldContent" type="nxs:content" />
   <xs:element name="beingRetrieved" type="xs:boolean" />
   <xs:element name="toBeRestored" type="xs:boolean" />
+  <xs:element name="downloadableUntil" type="xs:string" />
```

Comment:
Should be a `Date`. Take a look on [Here](https://github.com/nuxeo/nuxeo-retention/blob/master/nuxeo-retention/src/main/resources/schemas/record.xsd#L19).